pages/standard/[id].tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { GetServerSideProps } from 'next';
+import {
+    getDataSchema,
+    SchemaDataId,
+    DataSchemaData,
+} from '@htan/data-portal-schema';
+import { HtanNavbar } from '../../components/HtanNavbar';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Link from 'next/link';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons/faArrowLeft';
+import DataSchema from '../../components/DataSchema';
+import Footer from '../../components/Footer';
+
+interface ManifestProps {
+    schemaData: DataSchemaData;
+    requiredDependencies: DataSchemaData[];
+    schemaDataById: { [id: string]: DataSchemaData };
+}
+
+const Manifest: React.FC<ManifestProps> = ({
+    schemaData,
+    requiredDependencies,
+    schemaDataById,
+}) => {
+    return (
+        <>
+            <HtanNavbar />
+            <Container>
+                <Row style={{ marginBottom: 10 }}>
+                    <Col>
+                        <Link href="/standards">
+                            <a>
+                                <FontAwesomeIcon icon={faArrowLeft} />
+                                &nbsp; Back to Data Standards
+                            </a>
+                        </Link>
+                    </Col>
+                </Row>
+                <Row>
+                    <Col>
+                        <h1>{schemaData.attribute} Manifest</h1>
+                    </Col>
+                </Row>
+                <Row>
+                    <Col>
+                        <DataSchema
+                            schemaData={requiredDependencies}
+                            dataSchemaMap={schemaDataById}
+                        />
+                    </Col>
+                </Row>
+            </Container>
+            <Footer />
+        </>
+    );
+};
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    const { id } = context.params as { id: string };
```

Comment:
I think currently the id URL is in the form of `.../standard/bts:Therapy`. It would be better if we can remove the initial `bts:` and have URLs like `.../standard/Therapy` instead.